README.md

```diff
@@ -32,13 +32,16 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+*_Nikki_:  I've spent the last 15 years in the infrastructure world of IT.  Now that Nationwide is beginning to move to the cloud, having solid coding skills will be needed to transition to an Infrastructure as code cloud environment.
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+*_Nikki_:  I'm not much of a world traveler.  I'm a bit of a home body, but my wife and I have been trying to start taking vacations as of late.  We've been to Seattle, Colorado, and most recently, we went to Cancun.  Cancun was my favorite thus far because, well...rum.  I'd really like to go to Hawaii next, but am kind of dreading the flight.
```

Comment:
What intrigues you about Hawaii?  Beach?  Volcanoes? Rain forest?  Please add a little detail and update your PR.